src/Avalonia.Ide.CompletionEngine.DnlibMetadataProvider/Wrappers.cs

```diff
@@ -52,8 +52,8 @@ class TypeWrapper : ITypeInformation
         public IEnumerable<IPropertyInformation> Properties => _type.Properties
             //Filter indexer properties
             .Where(p =>
-                (p.GetMethod == null || p.GetMethod.Parameters.Count == (p.GetMethod.IsStatic ? 0 : 1))
```

Comment:
This effectively hides `internal` properties

Reply:
that's correct and intentional only public properties are to be completed, no private or internal. That's also the way wpf designer code completion works for me.

Reply:
It should be perfectly valid to access `internal` properties in the same assembly though.